Libraries/Utilities/MatrixMath.js

```diff
@@ -575,13 +575,12 @@ var MatrixMath = {
     // expose both base data and convenience names
     return {
       rotationDegrees,
-      perspective,
       quaternion,
       scale,
       skew,
       translation,
 
-      rotate: rotationDegrees[2],
+      perspective: perspective[2] != 0 ? -1/perspective[2] : 1280,
       scaleX: scale[0],
       scaleY: scale[1],
       translateX: translation[0],
```

Comment:
Why are the changes in this file needed? What's your test plan to make sure it doesn't break existing iOS behavior?